### ISSUE TITLE
Synthetic Dataset Analysis

### ISSUE THREAD

**Issue by lcorallo** [Uniba-dev-projects/forecasting-framework#1]
There is necessary to understand the behavior of the Forecasting Framework. It is composed by four machine learning models:
  1. Linear Regression
  2. Support Vector Regression
  3. Long Short Term Memory
  4. Fully Connected Neural Network
 
Each model will be tested with synthetic simple dataset in order to make some assumption. Parameter tuning can allow to delve into the context.

**Comment by lcorallo** [comment]
Closed with PR #2

**Comment by lcorallo** [comment]
Further analysis needs to be carried out.

- A new function y6(t) = sin(t) needs to be implemented.
- RMSE errors do not match what the graphs show.
- Distribution of train set and test set needs to be redone (random sampling)

**Comment by lcorallo** [comment]
Resolved with PR #3
